frontend/src/types.ts

```diff
@@ -2493,6 +2493,7 @@ export interface SurveyAppearance {
     widgetSelector?: string
     widgetLabel?: string
     widgetColor?: string
+    shuffleQuestions?: boolean
```

Comment:
reminder for posthog-js implementation, you've got to be very careful to make sure that the right response goes to the right question index, as otherwise it can mess up tracking / put the response for Q2 into Q1.

Reply:
sounds great, I'll keep that in mind.